libs/strings/string/src/lib.sw

```diff
@@ -173,17 +149,33 @@ impl String {
     }
 }
 
-// Uncomment when https://github.com/FuelLabs/sway/issues/3637 is resolved.
-// impl From<raw_slice> for String {
-//     fn from(slice: raw_slice) -> String {
-//         let mut bytes = Bytes::with_capacity(slice.number_of_bytes());
-//         bytes.buf.ptr = slice.ptr();
-//         Self {
-//             bytes
-//         }
-//     }
-
-//     fn into(self) -> raw_slice {
-//         asm(ptr: (self.bytes.buf.ptr(), self.bytes.len)) { ptr: raw_slice }
-//     }
-// }
+impl From<raw_slice> for String {
+    fn from(slice: raw_slice) -> String {
+        Self {
+            bytes: Bytes::from_raw_slice(slice),
```

Comment:
I assume this will be updated to use the `from` on `Bytes`

Reply:
Yes a new release is needed.